### ISSUE TITLE
Debug mode

### ISSUE THREAD

**Issue by Joelgullander** [M-Izadmehr/deadfile#4]
Should introduce a debug mode when running the scan, I got a huge project with .js & .ts files, it calculates the amount of assets & files, then just gets stuck & uncancellable. 

Is it compliant with typescript files?

**Comment by Joelgullander** [comment]
For example, it has been stuck on this line for an hour: 

247 imported assets found. /Users/xxxx/xxxx/xxxx.xxxx.ui/src/actions/cart/sync.js

**Comment by Joelgullander** [comment]
After more debugging it seems to stay in a endless loop at: 

```
  while (this.tok.value !== "from") {
    this.next(); // skip `export` to find `from`
  }
```

In src/ASTPlugins/parseImportFromExport.js

this.tok.value === undefined

**Comment by M-Izadmehr** [comment]
It should be fixed in major release V2.0.
